Composer/packages/ui-plugins/lg/src/LgField.tsx

```diff
@@ -36,10 +36,21 @@ const getInitialTemplate = (fieldName: string, formData?: string): string => {
 
 const LgField: React.FC<FieldProps<string>> = props => {
   const { label, id, description, value, name, uiOptions } = props;
-  const { designerId, currentDialog, lgFiles, shellApi, projectId, locale, userSettings } = useShellApi();
-
+  const { designerId, currentDialog, lgFiles, shellApi, projectId, locale, userSettings, data } = useShellApi();
+
+  let lgType = name;
+  const kind = data?.$kind;
+  if (kind) {
+    const [, schemaType] = kind.split('.');
+    if (schemaType === 'SendActivity') {
+      lgType = schemaType;
+    } else {
+      const nameCapitalized = name.charAt(0).toUpperCase() + name.slice(1);
+      lgType = `${schemaType}_${nameCapitalized}`;
+    }
+  }
   const singleLgRefMatched = value && value.match(/\$\{([\w-]+)(\(.*\))\}/);
```

Comment:
we can leverage the `LgTemplateRef` util (though it's not touched)

Reply:
Seems no need to make change on LgTemplateRef, here just passed a new lgType to it.